docs/eventing/features/authorization.md

```diff
@@ -0,0 +1,449 @@
+# Knative Eventing Authorization
+
+**Tracking issue**: [#7256](https://github.com/knative/eventing/issues/7256)
+
+## Overview
+
+Securing event delivery in Knative Eventing is essential to prevent unauthorized access. To enforce fine-grained control over event delivery, Knative Eventing introduces the `EventPolicy` custom resource, which allows users to specify which entities are authorized to send events to specific consumers within a namespace.
+
+## Prerequisites
+
+- [Eventing installation](./../../../install)
+- [`authentication-oidc`](./sender-identity.md) feature to be enabled.
```

Comment:
Doesn't this also require `transport-encyption` to be `strict`?

Reply:
AuthZ not directly. Only AuthN is not secure without it (that's described on the sender-identity feature page)

Reply:
But I can add a not there too